docs/src/howto.md

```diff
@@ -19,14 +19,15 @@ Background(1:100, ground)
 render(video; pathname="how_to.gif")
 ```
 
-## How can I avoid ambiguity errors with Luxor.jl?
+## Why are all my `Javis` functions undefined? 
 
-You might have noticed that [Luxor.jl](https://github.com/JuliaGraphics/Luxor.jl) provides the drawing functions that Javis is using.
-Javis is basically just one more package on top of it which provides functions to animate the static images you can create with Luxor.
-As one can't use Javis without using Luxor we decided to reexport all functions that Luxor exports. This means you don't have to and
-**shouldn't** call `using Luxor` as this will result in ambiguity errors. 
+If you have worked with the animation package [`Luxor.jl`](https://github.com/JuliaGraphics/Luxor.jl), you will be happy to that it provides all the drawing functions that `Javis` uses.
```

Comment:
*happy to **see** ?

Reply:
“animation package” is a bit of a stretch 😂

Reply:
Agree. I would call Javis an animation package and Luxor a drawing package.